Sources/NetworkService.swift

```diff
@@ -57,6 +57,17 @@ struct NetworkService: PushNotificationsRegisterable, PushNotificationsSubscriba
         self.setSubscriptions(interests: [])
     }
 
+    func track(userInfo: [AnyHashable : Any]) {
+        guard let publishId = PublishId(userInfo: userInfo).id else { return }
+        let timestamp = Date().milliseconds()
+
+        let bodyString = "{\"publishId\": \"\(publishId)\", \"timestampMs\": \"\(timestamp)\"}"
+        let body = Data(bodyString.utf8)
+
+        let request = self.setRequest(url: self.url, httpMethod: .POST, body: body)
+        self.networkRequest(request, session: self.session) { (response) in }
```

Comment:
I'll think about exception handling here.

Reply:
I think that we will need to retry this a couple of times before giving up. It could be done in a separate PR if you want.

Further down the road, we could be going for a write-ahead logging strategy to make sure we never miss out events (worse case scenario, we would get them later).

Reply:
Separate PR 👍